optimade/models/structures.py

```diff
@@ -84,7 +84,7 @@ def validate_chemical_symbols(cls, v):
     def validate_concentration(cls, v, values):
         if len(v) != len(values.get("chemical_symbols", [])):
             raise ValueError(
-                f"Length of concentration ({len(v)}) MUST equal length of chemical_symbols ({len(values.get('chemical_symbols', []))})"
+                f"Length of concentration ({len(v)}) MUST equal length of chemical_symbols ({len(values.get('chemical_symbols', 'Not specified'))})"
```

Comment:
Is  `'Not specified'` better than `None` here?

Reply:
I guess it's more informative if you're developing a server, which these messages are mainly for.